plugin.video.liveleak/addon.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.liveleak"
+    name="LiveLeak"
+    version="0.0.7"
+    provider-name="Botster">
+    <requires>
+        <import addon="xbmc.python" version="2.6.0"/>
```

Comment:
Corrected. But just FYI, http://kodi.wiki/view/Python_Development says:

> 8.1 Version
> Kodi comes with Python 2, version 2.6 or higher depending on platform. 

Thank you.